run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+# install dependencies
+mix deps.get
+
+# create and migrate database
+mix ecto.setup
+
+# install nodejs stuff
+cd assets && npm install
```

Comment:
```suggestion
cd assets && npm ci
```

Can you reset the package lock, and change this script to use ci? That we we don't accidentally upgrade the whole repo?